docker/prod-values.yml

```diff
@@ -71,8 +71,8 @@ daemon:
       command: ["/scripts/management_daemon.sh"]
       args: ["--delay", "300", "sync_techloan"]
 environmentVariables:
-  - name: AUTH_MODULE
-    value: oauth
+  - name: SPOTSEEKER_OAUTH_ENABLED
```

Comment:
since fake_oauth was never used, I replaced AUTH_MODULE with a boolean for oauth and all_ok. It also didn't need to have the long names for the settings since the files they refer to are no longer used for authentication.

Reply:
Would this ever actually get set to "false"?  I don't think this actually needs to be an ENV var, maybe just a test for ENV == "localdev"?